changelogs/fragments/880.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - lineinfile - Added missing variants for SNMPv3 authprotocol and privprotocol introduced by Zabbix 6
```

Please let's follow https://docs.ansible.com/ansible/latest/community/development_process.html#changelog-fragment-entry-format and change "scope":
```suggestion
  - zabbix_host - add missing variants for SNMPv3 authprotocol and privprotocol introduced by Zabbix 6
```